go/vt/sqlparser/testdata/select_cases.txt

```diff
@@ -9260,7 +9260,7 @@ INPUT
 select @@local.profiling;
 END
 OUTPUT
-select @@local.profiling from dual
+select @@`local.profiling` from dual
```

Comment:
This is incorrect. Just checked on MySQL - 
```mysql
mysql [localhost:8026] {msandbox} (test) > select @@local.profiling;
+-------------------+
| @@local.profiling |
+-------------------+
|                 0 |
+-------------------+
1 row in set, 1 warning (0.00 sec)

mysql [localhost:8026] {msandbox} (test) > select @@`local.profiling`;
ERROR 1193 (HY000): Unknown system variable 'local.profiling'
```

Reply:
nice catch